src/pages.c

```diff
@@ -833,6 +877,7 @@ pages_boot(void) {
 		os_pages_unmap(madv_free_page, PAGE);
 	}
 #endif
+	init_proces_madvise();
```

Comment:
Not handling the return value here?

Reply:
The only case I can think of when the init fails, is OOM during pidfd_open. Added return value check.